docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,312 @@
+======================
+HTTP client properties
+======================
+
+``http-client.https.hostname-verification``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Verifies that the server hostname matches the server certificate.
+
+``http-client.http2.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:**
+
+.. No default value?
+
+Enables the HTTP/2 transport.
+
+``http-client.connect-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``5s``
+
+``http-client.request-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``5m``
+
+``http-client.idle-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``1m``
+
+``http-client.max-connections``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1M``
```

Comment:
max-connections is the upper bound for connections to the client. I'm not sure this minimum value makes a lot of sense, and it might even be reasonable to not document a minimum value. The lower you go, the less reasonable it gets. You probably _could_ set it to 1, but your client would very easily break.